conslogging/conslogging.go

```diff
@@ -90,20 +91,37 @@ func (cl ConsoleLogger) Printf(format string, args ...interface{}) {
 
 // PrintBytes prints bytes directly to the console.
 func (cl ConsoleLogger) PrintBytes(data []byte) {
-	// TODO: This does not deal well with control characters, because of the prefix.
 	cl.mu.Lock()
 	defer cl.mu.Unlock()
-	if !bytes.Contains(data, []byte("\n")) {
-		// No prefix when it's not a complete line.
-		cl.w.Write(data)
-	} else {
-		adjustedData := bytes.TrimSuffix(data, []byte("\n"))
-		for _, line := range bytes.Split(adjustedData, []byte("\n")) {
-			cl.printPrefix()
-			cl.w.Write(line)
-			cl.w.Write([]byte("\n"))
+
+	output := []byte{}
```

Comment:
Should we do something like this here?

```suggestion
	output := make([]byte, 0, len(data))
```

... or what would be a better guess of the capacity?

Reply:
good plan, i'll make that change.